indico/modules/events/registration/fields/simple.py

```diff
@@ -371,3 +371,64 @@ def _indico_email(value):
                 raise ValidationError(_('Invalid email address'))
 
         return _indico_email
+
+
+class AccompanyingPersonsField(RegistrationFormBillableField):
```

Comment:
I think this isn't a "simple" field... maybe something for a separate file (e.g. `accompanying.py`)?